### PR TITLE
helm 3.16.0 and kubectl 1.31.1

### PR DESCRIPTION
**Pull request by jnoordsij**
See https://github.com/helm/helm/releases/tag/v3.16.0 and https://github.com/kubernetes/kubernetes/releases/tag/v1.31.1

*Edit:* for some reason not apparent to me, fetching the `kubectl` binary from https://storage.googleapis.com/kubernetes-release/ does not work yet. I could not find a deprecation notice; however there may be some delay in this distribution?

Anyways, when checking official docs and release notes, both seem to suggest using https://dl.k8s.io/ as source (see e.g. https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/ or https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.31.md#downloads-for-v1311), therefore I propose to change this accordingly.

### CODE REVIEW

**Comment by jnoordsij** [comment]
Ah good point! Just fixed it (added new tag and fixed previous one).

File: README.md [hunk 1]
```diff
@@ -6,7 +6,8 @@
 
 Supported tags and release links
 
-* [3.15.4](https://github.com/dtzar/helm-kubectl/releases/tag/3.15.3) - helm v3.15.4, kubectl v1.30.3, alpine 3.20
+* [3.16.0](https://github.com/dtzar/helm-kubectl/releases/tag/3.16.0) - helm v3.16.0, kubectl v1.31.1, alpine 3.20
+* [3.15.4](https://github.com/dtzar/helm-kubectl/releases/tag/3.15.4) - helm v3.15.4, kubectl v1.30.3, alpine 3.20
```
Comment (gaby) [inline README.md:10]: You can remove 3.15.x here, only leave 3.16.x

File: .env [hunk 2]
```diff
@@ -1,3 +1,3 @@
-KUBE_VERSION=1.30.3
-HELM_VERSION=3.15.4
+KUBE_VERSION=1.31.1
+HELM_VERSION=3.16.0
 YQ_VERSION=4.44.3
```
Comment (gaby) [inline .env:3]: Any updates available for YQ ?

**Review by gaby — CHANGES_REQUESTED** [review]
Lgtm, small change

File: .env (hunk 2 above)
Comment (jnoordsij) [reply .env:3]: Nope: https://github.com/mikefarah/yq/releases

**Pull request merged**